docs/tango-vmms.md

```diff
@@ -2,7 +2,7 @@ This page documents the interface for Tango's Virtual Machine Management Systems
 
 See [the vmms directory](https://github.com/autolab/Tango/tree/master/vmms) in Tango for example implementations. 
 
-We currently provide [Docker VMMS](/tango/#docker-vmms-setup) and the [Amazon EC2 VMMS](tango/#amazon-ec2-vmms-setup) interfaces. The following API is for developers who intend to use a different kind of VMMS for Tango's autograding purposes.
+We currently provide [Docker VMMS](/tango-vmms/#docker-vmms-setup) and the [Amazon EC2 VMMS](/tango-vmms/#amazon-ec2-vmms-setup) interfaces. The following API is for developers who intend to use a different kind of VMMS for Tango's autograding purposes.
```

Comment:
@victorhuangwq Hi Victor! Is there any particular reason why you didn't include the Docker and Amazon EC2 VMMS setup in tango-vmms.md in your PR but put them in tango.md? It does say above that this page offers instructions for setting up VMMSs.

Reply:
There were two reason I moved the Docker and Amazon EC2 VMMS into tango.md
- the set up instructions were not exactly "Tango VMMS API" per say?
- at the same time, people who were installing Tango has to choose one of them to be set up, so it made sense to be on tango.md

Reply:
Then I think it makes more sense to put such information in installations/tango.md? Since the refactored tango.md is more of an overview?

Reply:
okay i think that makes sense.